hello.html

```diff
@@ -8,6 +8,25 @@
 
     <script type="text/javascript">
 
+    /* *********************************
+     * Prepare for bridge initialization
+     * *********************************/
+     
+     // Launched stand-alone, startApp() on the bridge ready event
+     document.addEventListener('WebViewJavascriptBridgeReady', function(event) {
```

Comment:
Let's also run the checks for whether we are running stand alone or embedded from outside the `window.onload`.

Same fix will need to be added to the other files too, thank you!